### ISSUE TITLE
Ubuntu 20.04 Unable to find NET_DM family, dropwatch can't work, kernel already configured with CONFIG_NET_DROP_MONITOR=y

### ISSUE THREAD

**Issue by jmck111** [nhorman/dropwatch#91]
I have installed dropwatch using snap on Ubuntu 20.04. Running Kernel is 5.4.0-200-generic

When I run: dropwatch -l kas

 I get:
Unable to find NET_DM family, 
dropwatch can't work

I have checked the kernel var CONFIG_NET_DROP_MONITOR and its already set to y.
Any ideas?
Thanks.

**Comment by nhorman** [comment]
I'm guessing its some kernel misconfiguration.  I just tried on 22.04 with kernel 5.15.0-122 and it works fine.  Perhaps you're looking at the config for a newer kernel but running a version that doesn't have it enabled.  Suggest opening an issue with ubuntu.

**Comment by jmck111** [comment]
Thanks for the quick reply. Hmm seems to be the correct Kernel I'm checking. I'll do some more digging.

**Comment by nhorman** [comment]
Did you figure out what was going on here?

**Comment by jmck111** [comment]
No. As far as I remember I did check the kernel configuration and the flags required were set. I ended up making a complete reinstall of the system to Ubuntu 22 and it worked then. Not sure how I ended up in the original situation.

**Comment by AndySchroder** [comment]
I had a similar problem with the snap version. I compiled from source and it worked fine.
